### PR TITLE
Remove user profile

### PR DESCRIPTION
**Pull request by sufeiiz**
- removed user profile code
- removed 3rd slide regarding profiles on Welcome ViewPager
- replaced drawer profile with droidcon logo

### CODE REVIEW

**Review by candidcold — APPROVED** [review]
http://68.media.tumblr.com/6c2359a914e19b4b3bf60adbd17c7341/tumblr_miuppeKb2Z1qzc2azo1_500.gif

File: app/src/main/kotlin/co/touchlab/droidconandroid/WelcomeActivity.kt [hunk 1]
```diff
@@ -32,7 +34,7 @@ class WelcomeActivity : AppCompatActivity() {
         setContentView(R.layout.activity_welcome)
 
         val short = intent.getBooleanExtra(SHORT, false)
-        val lastIndex = if (short) 1 else 2
+        val lastIndex = if (short) LAST_INDEX_SHORT else LAST_INDEX
```
Comment (samhill303) [inline app/src/main/kotlin/co/touchlab/droidconandroid/WelcomeActivity.kt]: This if doesnt appear to do anything

File: app/src/main/kotlin/co/touchlab/droidconandroid/ui/WelcomePagerAdapter.kt [hunk 2]
```diff
@@ -21,16 +21,17 @@ class WelcomePagerAdapter(fm: FragmentManager, val short: Boolean) : FragmentPag
...
-//                3 -> return SponsorWelcomeFragment()
-//                4 -> return Sponsor2WelcomeFragment()
             }
             throw IllegalStateException("Too many fragments")
         }
     }
 
     override fun getCount(): Int {
-        return if (short) 2 else 3
+        return if (short) SHORT_COUNT else FRAGMENT_COUNT
```
Comment (samhill303) [inline app/src/main/kotlin/co/touchlab/droidconandroid/ui/WelcomePagerAdapter.kt]: Same here

**Comment by sufeiiz** [comment]
@samhill303 removed "SHORT" and corresponding sponsor fragments

**Review by samhill303 — APPROVED** [review]

**Pull request merged**